bundles/org.openhab.binding.etherrain/README.md

```diff
@@ -0,0 +1,61 @@
+# Etherrain Binding
+
+The Etherrain binding is used to control a sprinkler controller from Quicksmart:
+
+http://www.quicksmart.com/qs_etherrain.html
+
+The API exposed by the controller is fairly robust, but it is specific. 
+The binding attempts to map this API to an openhab thing while perserving the functionality.
+
+## Overview
+
+An Etherrain controller can control 7-8 zones (depending on model) and can also report on the status of a rain sensor.
+The controller itself accepts a single command that contains an initial delay, and an on-time for each of the zone. 
+Once this execute command is sent, the controller will, first, wait the initial delay time, then cycle through each zone and turn it on the amount of time specified. 
+The binding exposes the rain sensor as a contact as well as the operating status of of the controller. 
+
+
+## Supported Things
+
+The bridge represents a physical Etherrain controller and contains all channels need to control it. 
+
+## Discovery
+
+The binding will automatically discover Etherrain controllers when a thing is added.
```

Comment:
But above you say that it supports just controllers being the bridge? What are the things then? It seems that this part of the README is either outdated or you misunderstood the concept of a bridge, what you have is a Thing (your binding does not have a bridge), so you can remove all references to bridge in the README